jeansantos38/stf-kick-off
# Topics/jeansantos38@gmail.com/desktopdemo

## QA Checkpoint
| Item | Details | Notes
| ------ | ------ | ------ |
| 1|  Results from the test run are clearly highlighted in PR and US | e.g.: Planned manual/automated tests were done/executed and are all passing|
| 2 | Confirmation that all existent Unit tests were run and passed successfully. Confirmation that unit coverage meets at least 80% | Yes, all unit tests are passing, see details below in **codecov** |
| 3 | Added code was also covered by unit tests and current coverage was not decreased. Info: There’s already a mechanism in GitHub that blocks the PR in case it’s less than 80% or even if still above but was decreased after adding new code. | Yes, current coverage was not decreased, see details below in **codecov** |
| 4 | Known bugs were created in Jira and require a comment from the owner approving the release with those.  | eg1: No bugs identified during tests ###  eg2: Low priority bug [SW3DMAG-198](https://hp-ibdv.atlassian.net/browse/SW3DMAG-198) found and won't be fixed, for release 123ABC, accordingly owner approval available here [SW3DMAG-279](https://hp-ibdv.atlassian.net/browse/SW3DMAG-279).  |

### Planned Tests and details
| Test name | Details
| ------ | ------ |
| TCX+1 - Any reference to the test, e.g. respective gitHub repo README, TestRail, etc.| Even the test is well described somewhere, additional information such as STACK, Status (pass/fail/skipped), custom environment setup/test files/parts used, that are non mentioned in documentation, should be provided here. |
| [TCX+2](https://www.google.com/) | Stack: Dev, Result: PASS, Env. details:  Using a reduced bed size (x,y,z) due to memory (ram) constraints, to limit the max number of parts in 3 per bucket|
